packages/backend/src/plugins/permission.ts

```diff
@@ -47,13 +48,36 @@ class AccessPolicy implements PermissionPolicy {
 }
 
 export default async function createPlugin(
-    env: PluginEnvironment,
+  env: PluginEnvironment,
 ): Promise<Router> {
-    return await createRouter({
-        config: env.config,
-        logger: env.logger,
-        discovery: env.discovery,
-        policy: new AccessPolicy(),
-        identity: env.identity,
-    });
+  return await createRouter({
+    config: env.config,
+    logger: env.logger,
+    discovery: env.discovery,
+    policy: new AccessPolicy(),
+    identity: env.identity,
+  });
 }
+
+/**
+ * Read user group from config
+ */
+let group = '';
+export async function main() {
```

Comment:
Could we rename this function to readUserGroupFromConfig and get rid of the comment above?

Reply:
Not now, i will explain separately the workflow.